### DISCUSSION TITLE
Pinia referenced in Layout and content page simultaneously will crash

### DISCUSSION THREAD

**Discussion by yeechutngeow** [quasarframework/quasar#14649] [General - Components / Directives / etc]
Hi,

I just found out that if I try to reference a Pinia store in the Layout while the content page is also referencing the same store then the store won't work anymore. 

The situation is a bit more complex and perplexing:
- SignIn.vue (simple layout) 
    calls store action to save the user's credential.
    $router.push('mycontent')
- MyContent.vue (with  MainLayout.vue) both references the Piana store

The failure happens in SignIn.vue. If I removes the reference to the Piana store in MainLayout.vue then everything works as expected.

**Comment by MilosPaunovic** [comment]
This doesn't seem like it should happen, but we'll need a code reproduction to take a closer look at the issue.

**Reply by rstoenescu** [reply]
There's definitely something in your app that is off. But we can't help without a reproduction.

**Reply by yeechutngeow** [reply]
My apology - it was a coding error. I had a state named loggedIn, and an action logIn()..
I set logIn = true - overwriting the function to a boolean.... sigh. Sorry again.
